### PR TITLE
fix phar build

### PR DESCRIPTION
**Pull request by smoench**

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -28,15 +28,15 @@ jobs:
                     coverage: "none"
                     extensions: "${{ env.REQUIRED_PHP_EXTENSIONS }}"
                     php-version: "${{ matrix.php-version }}"
-                    tools: composer:v2
+                    tools: composer:2.1
```
Comment (OskarStark) [inline .github/workflows/build.yaml:31]: Is this change needed?

Comment (smoench) [reply .github/workflows/build.yaml:31]: Yes, it seems box has some issues with composer 2.2.x

Comment (OskarStark) [reply .github/workflows/build.yaml:31]: Ok 👍

**Comment by OskarStark** [comment]
Ready to merge? :-)

**Comment by OskarStark** [comment]
Thank you
